app/models/order.rb

```diff
@@ -3,6 +3,8 @@ class Order < ApplicationRecord
   has_many :order_items, dependent: :destroy
   has_many :products, through: :order_items
 
+  accepts_nested_attributes_for :order_items
```

Comment:
accepts_nested_attributes_for nó có các option gì e nhỉ

Reply:
Nó có 4 option:
  -Allow_destroy: True hoặc false, nếu là true thì có thể xóa record nào chưa key _destroy và giá trị là true.
  -Reject_if: Cho phép chỉ định một Proc hoặc một Symbol trỏ vào một phương thức để kiểm tra một bản ghi có được phép tạo ra hay không.
  -Limit Cho phép chỉ định số lượng tối đa của các record liên quan có thể được xử lý với các thuộc tính lồng nhau. :limit option chỉ áp dụng cho quan hệ one-to-many.
  -Update_only Cho phép chỉ định một record chỉ có thể được update thôi. Một record mới được tạo ra khi ko có record nào hiện có. :update_only chỉ hoạt động cho quan hệ one-to-one.